BitwardenAutoFillExtension/CredentialProviderViewController.swift

```diff
@@ -90,6 +90,15 @@ class CredentialProviderViewController: ASCredentialProviderViewController {
         default:
             break
         }
+
+        guard #available(iOSApplicationExtension 18.0, *) else {
+            return
+        }
+
+        if let otpRequest = credentialRequest as? ASOneTimeCodeCredentialRequest,
+           let otpIdentity = otpRequest.credentialIdentity as? ASOneTimeCodeCredentialIdentity {
+            provideOTPCredentialWithoutUserInteraction(for: otpIdentity)
+        }
```

Comment:
🎨 Would it make sense to put this in the default case of the switch statement to keep everything in the switch? Also applies to `prepareInterfaceToProvideCredential`.

Reply:
Not sure what would we gain given that we wouldn't leverage the case feature and the code will be even more nested. However if you have a strong opinion about it I can do the change no problem at all.
Or were you meaning to call a function from the `default` which has this code in it?

Sadly, dealing with `#available` is a bit messy and when dealing with multiple logic per OS version I'm not sure if I like having methods with suffixes like `myAwesomeMethodIOS17`, `myAwesomeMethodIOS18(...)`, so on; although maybe it could be a solution that is pretty self-explanatory.

Reply:
I think it initially reads like something that happens after the switch currently and could apply to any of the above cases, but it's really just another case that we're looking for based on `credentialRequest`. I agree the available checks aren't great. I'm fine keeping it as-is if you prefer that!

```
        default:
            if #available(iOSApplicationExtension 18.0, *),
               let otpRequest = credentialRequest as? ASOneTimeCodeCredentialRequest,
               let otpIdentity = otpRequest.credentialIdentity as? ASOneTimeCodeCredentialIdentity {
                provideOTPCredentialWithoutUserInteraction(for: otpIdentity)
            }
```